AstSemantics.md

```diff
@@ -200,13 +200,13 @@ Out of bounds accesses trap.
 
 ### Resizing
 
-In the MVP, linear memory can be resized by a `grow_memory` operator. This
-operator requires its operand to be a multiple of the WebAssembly page size,
+In the MVP, linear memory can be resized by a `grow_memory` operator. The
+operand to this operator is in units of the WebAssembly page size,
 which is 64KiB on all engines (though large page support may be added in 
 the [future](FutureFeatures.md#large-page-support)).
 
- * `grow_memory` : grow linear memory by a given unsigned delta which
-    must be a multiple of the page size. Return the previous memory size.
+ * `grow_memory` : grow linear memory by a given unsigned delta of pages.
+    Return the previous memory size in bytes.
```

Comment:
There seems to be consensus to change the argument to pages so perhaps that could land first and then you could champion a change of the memory_size to pages too in a follow up PR, or do people want it changed here too?